Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -1048,6 +1048,77 @@ static gboolean webkitWebViewAccumulatorObjectHandled(GSignalInvocationHint*, GV
     return !object;
 }
 
+#if PLATFORM(GTK) && USE(GTK4)
+static GdkEvent* gValueGetEvent(const GValue* value)
+{
+    g_return_val_if_fail(G_VALUE_HOLDS(value, GDK_TYPE_EVENT), NULL);
+
+    return reinterpret_cast<GdkEvent*>(value->data[0].v_pointer);
+}
+
+typedef gboolean (*ContextMenuCallback) (gpointer, WebKitContextMenu*, GdkEvent*, WebKitHitTestResult*, gpointer);
+
+static void webkitWebViewContextMenuMarshal(GClosure* closure, GValue* returnValue, guint nParams, const GValue* params, gpointer, gpointer marshalData)
+{
+    GCClosure* cc = (GCClosure *) closure;
```

Comment:
```suggestion
    auto* cc = static_cast<GCClosure*>(closure);
```

No C-style casts unless C++ casts are impossible (very rare). Also, no point in writing out the name of the type twice. Also, your * must hang on the type.

Reply:
Ah yeah. This code was based on glib-genmarshal output so the starting point was C. :)